src/deadline_test_fixtures/fixtures.py

```diff
@@ -438,7 +445,10 @@ def worker_config(
         with src_path.open(mode="w") as f:
             json.dump(service_model.model, f)
 
-        dst_path = posixpath.join("/tmp", src_path.name)
+        if operating_system == "AL2023":
```

Comment:
All this branching seems like an opportunity to create a subclass for each variant. Maybe for future work.